iceberg/src/test/scala/org/apache/spark/sql/delta/ConvertToIcebergSuite.scala

```diff
@@ -123,21 +128,160 @@ class ConvertToIcebergSuite extends QueryTest with Eventually {
         withDefaultTablePropsInSQLConf {
           deltaSpark.range(10).write.format("delta")
             .option("path", testTablePath)
+            .option("delta.enableIcebergCompatV2", "true")
```

Comment:
None of these tests passed for me (even before my change) without setting this so I'm fixing it in this PR